services/user.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+
+def create_user(username: str, password: str, **kwargs) -> User:
+    return User.objects.create_user(
+        username=username,
+        password=password,
+        **kwargs
+    )
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.filter(id=user_id).first()
+
+
+def update_user(user_id: int, **kwargs) -> User:
```

Comment:
The return type annotation for `update_user` should be `Optional[User]` to indicate that the function may return `None` if the user is not found.